### DISCUSSION TITLE
Renewal of domain, registered throuogh Vercel, after it expired

### DISCUSSION THREAD

**Discussion by tunnckoCore** [vercel/community#5007] [Help] [answered]
### Summary

Hey there. Just realized my main domain expired (which is registered and still on Vercel), because i forgot to update the billing details with my new cards.

Many of my emails are linked/using it (routing them through AnonAddy to Tutanota), so renewing is needed ASAP. 

I couldn't find any other place to ask chat or support or anything.

I don't see how to renew either. I looked around the domains dashboard and every through the domains search field where it's still shown as taken.

What to do?

### Steps to Reproduce

My domain expired.

**Comment by amyegan** [comment, marked as answer]
Hey @tunnckoCore. I understand you may not feel comfortable revealing the domain here. Please [submit a case](https://vercel.com/help) to the Customer Success Team with `Domain Renewals` selected as the problem area. That will get you in private contact with the team so you can safely discuss account, domain, and billing details. Be sure to include the domain in your message so the team can check on it as quickly as possible.

**Reply by tunnckoCore** [reply]
okay, thanks :100: i might missed the `Domain Renewals` from the drop down, haha..
